docs/.vitepress/theme/Components/Content/Button.vue

```diff
@@ -1,14 +1,34 @@
 <template>
-	<a :class="`btn ${color}`" :href="link"><slot /></a>
+	<component :is="link ? `a` : `button`" class="btn" :class="classes" :href="link">
+		<component v-if="icon" class="btn-icon" :class="[iconColor]" :is="icon" />
+		<div>
+			<slot />
+		</div>
+		<component v-if="iconRight" class="btn-icon" :class="[iconColor]" :is="iconRight" />
+	</component>
 </template>
 
-<script setup lang="ts">
-import { defineProps } from 'vue'
+<script setup="props" lang="ts">
```

Comment:
You can revert the change here. `<script setup lang="ts">` is enough. Importing props this way is no longer needed

Reply:
Done.

Really like the `setup` syntax. Never knew it was a thing until going through code.